### ISSUE TITLE
Bahamut

### ISSUE THREAD

**Issue by GoogleCodeExporter** [tel8618217223380/game-golem#38]
```
Okay, I've added the Warrior buttons to the list, but I've noticed that the 
health for Bahamut uses a different background image.  (nm_red.jpg)  Not 
only that but the "fortification" for Bahamut uses nm_green.jpg.

And that's still not all.  There is a new thing called strengthen or 
strength.  It is basically the max you can fortify.  I have the button to 
boost strength as a warrior.  (Listed in the code now.)

So I've added the strengthen button and the bash (attack) button to the 
lists.  I've also created lists for the health bar and the fortification 
bar images.  I also threw the dispel bar image into an array for 
completeness.  The health, dispel, and fortification parsing are not 
currently setup to use the arrays yet.

For the strength thing, we're going to need to parse the nm_green.jpg 
parent's width as well as the grandparent's width.  The parent's width is 
the sub-fortification amount.  The grandparent's width is the strength 
amount.  Combine the two and you have the total-fortification.

Not sure how you want to handle the "fortify before attacking" checkbox.  
The real health (the one that gives bonuses or penalties) is the health 
percentage times the strength percentage.  But there is no way that one 
person can increase both.  Clerics increase fortification.  Warriors 
increase strength.

I would say clerics heal/fortify until sub-fortification percentage is 100% 
or total-fortification percentage (sub-fort x strength) is whatever is set 
in the options.

I would say warriors strengthen until the strength is whatever is set in 
the options.


So in summary:

1) We need to use the health, dispel, and fortification bar arrays for 
parsing.
2) We need to handle parsing strength and fortification when present.
3) We need to decide what to do based on our class.
```

Original issue reported on code.google.com by `brianbar...@gmail.com` on 22 Apr 2010 at 7:13
